src/main/java/com/hybris/builder/CLIRunner.java

```diff
@@ -671,11 +758,20 @@ protected void lookupMavenBinary() throws IOException
         }
     }
 
+    /**
+     * Check if the operating system is a windows machine.
+     * @return true if the operating system is a windows machine.
+     */
     private boolean isWindows()
     {
         return System.getProperty("os.name").contains("indows");
     }
 
+    /**
+     * Check if maven binary is installed independent of the operating system.
+     * @param filename
+     * @return the absolute path to the maven binary.
```

Comment:
Machst du irgendwas mit denen? Sagen dir die Meldungen was?



From: hardl [mailto:notifications@github.com]
Sent: Donnerstag, 1. Oktober 2015 16:12
To: SAP/builder-cli
Subject: Re: [builder-cli] TSK-2838 (#2)



In src/main/java/com/hybris/builder/CLIRunner.java<https://github.com/SAP/builder-cli/pull/2#discussion_r40917101>:

>      private boolean isWindows()
>      {
>          return System.getProperty("os.name").contains("indows");
>      }
>
> +    /**
> +     * Check if maven binary is installed independent of the operating system.
> +     * @param filename
> +     * @return the absolute path to the maven binary.

not bound to maven binary

—
Reply to this email directly or view it on GitHub<https://github.com/SAP/builder-cli/pull/2/files#r40917101>. <https://github.com/notifications/beacon/ANZtefQzC3O1Kvg6wvXpatABBssv-4Deks5o3TaygaJpZM4GHR_g.gif>